src/ansi-c/expr2c.cpp

```diff
@@ -41,6 +41,21 @@ Author: Daniel Kroening, kroening@kroening.com
 #include "expr2c.h"
 #include "expr2c_class.h"
 
+namespace {
+  std::string escape(const std::string &str)
+  {
+    std::string result;
+    result.reserve(str.size());
+    for(const auto &ch : str)
+    {
+      if(ch=='"')
```

Comment:
@LAJW  I created some regression tests for `\u000a` and `\u000d`. They are here: https://github.com/diffblue/test-gen/tree/allredj/regression/DisallowedChars
You can integrate them to the test-gen PR if you plan to fix that. Otherwise a new issue for these characters should probably be created.